experiments/gradle/01-validate-incremental-build.sh

```diff
@@ -0,0 +1,291 @@
+#!/usr/bin/env bash
```

Comment:
When running without extra args, the following empty quotes are printed on the cmd line (they should not):

```
Running first build:
./gradlew '' --no-build-cache clean build -Dscan.tag.exp1-gradle -Dscan.tag.607d3f63
```